clicommand/artifact_download.go

```diff
@@ -22,17 +22,22 @@ Description:
 
 Example:
 
-   $ buildkite-agent artifact download "pkg/*.tar.gz" . --build xxx
+   $ buildkite-agent artifact download --build xxx "pkg/*.tar.gz" .
 
    This will search across all the artifacts for the build with files that match that part.
    The first argument is the search query, and the second argument is the download destination.
 
    If you're trying to download a specific file, and there are multiple artifacts from different
    jobs, you can target the particular job you want to download the artifact from:
 
-   $ buildkite-agent artifact download "pkg/*.tar.gz" . --step "tests" --build xxx
+   $ buildkite-agent artifact download --step "tests" --build xxx "pkg/*.tar.gz" .
 
-   You can also use the step's jobs id (provided by the environment variable $BUILDKITE_JOB_ID)`
+   You can also use the step's jobs id (provided by the environment variable $BUILDKITE_JOB_ID)
```

Comment:
This feels like a typo?

```suggestion
   You can also use the step's job id (provided by the environment variable $BUILDKITE_JOB_ID)
```